### ISSUE TITLE
Unexpected behavior with fill(none) 

### ISSUE THREAD

**Issue by hoomanv** [influxdata/influxdb#5404]
My influxdb version: 0.10.0_nightly_72c6a51-1

Here is my sample data
```
> select * from QueueEvent
name: QueueEvent
----------------
time                    operation       queue   queueLength     realm   token   user    waitTime
1453309133951183524     1               1       1               1       A1      3
1453309134107323886     1               1       2               1       A2      3
1453309134232486449     1               1       3               1       A3      3
1453309134545157506     2               1       2               1       A1      4       0
1453309134670908848     3               1                       1       A1      4
1453309134717119367     1               1       3               1       A1      3
1453309134857480188     2               1       2               1       A2      4       0
1453309134967231854     3               1                       1       A2      4
1453309135076390861     2               1       1               1       A3      4       0
1453309135170021788     2               1       0               1       A1      4       0
1453309135326495116     3               1                       1       A1      4
1453309137572828846     1               1       1               2       C100    2
1453309137634689975     1               2       1               1       B10     3
1453309137806418512     2               2       0               1       B10     4       0
1453309137931373970     1               1       1               1       A1      3
1453309138056163705     2               1       0               1       A1      4       0
1453309138181920931     1               2       1               1       B11     3
1453309138306042911     1               1       1               1       A2      3
1453309138431029729     1               1       2               1       A3      3
1453309138619481684     2               1       0               2       C100    2       1
1453309138666355485     2               1       1               1       A2      4       0
1453309138775515158     2               1       0               1       A3      4       0
1453309138900106850     2               2       0               1       B11     4       0
1453309139103052450     1               1       1               2       C101    2
1453309139119448870     1               1       1               1       A1      3
1453309139259630884     2               1       0               1       A1      5       0
1453309139385695481     1               2       1               1       B12     3
1453309139510364091     2               2       0               1       B12     5       0
1453309139635319564     1               2       1               1       B10     3
1453309139760457141     1               1       1               1       A2      3
1453309139885871379     1               1       2               1       A3      3
1453309140009474114     1               2       2               1       B11     3
1453309140041048475     2               1       0               2       C101    2       0
1453309140088817438     2               2       1               1       B10     5       0
1453309140228990898     2               1       1               1       A2      5       0
1453309140244014387     2               1       0               1       A3      5       0
1453309140259682341     2               2       0               1       B11     5       0
1453309140416996032     1               1       1               1       A1      3
1453309140541016379     1               2       1               1       B12     3
1453309140666611631     1               1       1               2       C102    2
1453309140681623607     1               1       2               2       C100    2
1453309145702139437     1               1       2               1       A2      3
1453309145827113811     1               2       2               1       B10     3
1453309152171199228     4               2       1               1       B12
1453309152171766913     4               1       1               1       A1
1453309152281054977     2               1       0               1       A2      5       6
1453309152406600887     2               2       0               1       B10     5       6
1453309152531218044     2               1       1               2       C102    2       11
1453309156173414613     4               1       0               2       C100
```
realm,queue,user,operation are tags
queueLength,waitTime,token are fields

And here are the results of two similar queries, one with fill(none) and one without
```
> select count(queueLength),max(queueLength),min(queueLength),sum(queueLength) from QueueEvent where time > 1453300000000000000 group by time(1h)

name: QueueEvent
----------------
time                    count   max     min     sum
1453298400000000000     0
1453302000000000000     0
1453305600000000000     46      3       0       45
1453309200000000000     0
1453312800000000000     0

> select count(queueLength),max(queueLength),min(queueLength),sum(queueLength) from QueueEvent where time > 1453300000000000000 group by time(1h) fill(none)
name: QueueEvent
----------------
time    count   max     min     sum

>
```

The one with fill(none) returns nothing but it should have returned the following
```
1453305600000000000     46      3       0       45
```

**Comment by hoomanv** [comment]
Any ideas on this?

**Comment by rossmcdonald** [comment]
@hoomanv Which OS and version? Also, which storage engine are you using?

If possible, I would recommend upgrading to `0.10.0-beta1`, which has a few extra fixes that are not in your version.

https://s3.amazonaws.com/influxdb/influxdb-0.10.0-0.beta1.x86_64.rpm
https://s3.amazonaws.com/influxdb/influxdb_0.10.0-0.beta1_amd64.deb

**Comment by hoomanv** [comment]
@rossmcdonald
CentOS 6.7, I'm not sure what you mean by storage engine?
I'll try installing 0.10.0-beta soon.
Also below is the write protocol used to generate the above sample points. The times are different but in the same order.
```
QueueEvent,operation=1,queue=1,realm=1,user=3 queueLength=1i,token="A1" 1453801849287949499
QueueEvent,operation=1,queue=1,realm=1,user=3 queueLength=2i,token="A2" 1453801849443748187
QueueEvent,operation=1,queue=1,realm=1,user=3 queueLength=3i,token="A3" 1453801849459694980
QueueEvent,operation=2,queue=1,realm=1,user=4 queueLength=2i,waitTime=0i,token="A1" 1453801849537039366
QueueEvent,operation=3,queue=1,realm=1,user=4 token="A1" 1453801849662885217
QueueEvent,operation=1,queue=1,realm=1,user=3 queueLength=3i,token="A1" 1453801849693431996
QueueEvent,operation=2,queue=1,realm=1,user=4 queueLength=2i,waitTime=0i,token="A2" 1453801849833324225
QueueEvent,operation=3,queue=1,realm=1,user=4 token="A2" 1453801849943002951
QueueEvent,operation=2,queue=1,realm=1,user=4 queueLength=1i,waitTime=0i,token="A3" 1453801849943133135
QueueEvent,operation=2,queue=1,realm=1,user=4 queueLength=0i,waitTime=0i,token="A1" 1453801850130344935
QueueEvent,operation=3,queue=1,realm=1,user=4 token="A1" 1453801850145681073
QueueEvent,operation=1,queue=1,realm=2,user=2 queueLength=1i,token="C100" 1453801852221542057
QueueEvent,operation=1,queue=2,realm=1,user=3 queueLength=1i,token="B10" 1453801852252190180
QueueEvent,operation=2,queue=2,realm=1,user=4 queueLength=0i,waitTime=0i,token="B10" 1453801852315244648
QueueEvent,operation=1,queue=1,realm=1,user=3 queueLength=1i,token="A1" 1453801852440522631
QueueEvent,operation=2,queue=1,realm=1,user=4 queueLength=0i,waitTime=0i,token="A1" 1453801852549975083
QueueEvent,operation=1,queue=2,realm=1,user=3 queueLength=1i,token="B11" 1453801852674921289
QueueEvent,operation=1,queue=1,realm=1,user=3 queueLength=1i,token="A2" 1453801852689221910
QueueEvent,operation=1,queue=1,realm=1,user=3 queueLength=2i,token="A3" 1453801852814150566
QueueEvent,operation=2,queue=1,realm=2,user=2 queueLength=0i,waitTime=0i,token="C100" 1453801852954412016
QueueEvent,operation=2,queue=1,realm=1,user=4 queueLength=1i,waitTime=0i,token="A2" 1453801852986217741
QueueEvent,operation=2,queue=1,realm=1,user=4 queueLength=0i,waitTime=0i,token="A3" 1453801852986238137
QueueEvent,operation=2,queue=2,realm=1,user=4 queueLength=0i,waitTime=0i,token="B11" 1453801853001274238
QueueEvent,operation=1,queue=1,realm=2,user=2 queueLength=1i,token="C101" 1453801853157147917
QueueEvent,operation=1,queue=1,realm=1,user=3 queueLength=1i,token="A1" 1453801853173364488
QueueEvent,operation=2,queue=1,realm=1,user=5 queueLength=0i,waitTime=0i,token="A1" 1453801853313088153
QueueEvent,operation=1,queue=2,realm=1,user=3 queueLength=1i,token="B12" 1453801853422244856
QueueEvent,operation=2,queue=2,realm=1,user=5 queueLength=0i,waitTime=0i,token="B12" 1453801853532568015
QueueEvent,operation=1,queue=1,realm=1,user=3 queueLength=1i,token="A2" 1453801853656874842
QueueEvent,operation=1,queue=2,realm=1,user=3 queueLength=1i,token="B10" 1453801853656959664
QueueEvent,operation=1,queue=1,realm=1,user=3 queueLength=2i,token="A3" 1453801853781784907
QueueEvent,operation=1,queue=2,realm=1,user=3 queueLength=2i,token="B11" 1453801853797610143
QueueEvent,operation=2,queue=1,realm=2,user=2 queueLength=0i,waitTime=0i,token="C101" 1453801853902354613
QueueEvent,operation=2,queue=2,realm=1,user=5 queueLength=1i,waitTime=0i,token="B10" 1453801853919632501
QueueEvent,operation=2,queue=1,realm=1,user=5 queueLength=1i,waitTime=0i,token="A2" 1453801854027368536
QueueEvent,operation=2,queue=1,realm=1,user=5 queueLength=0i,waitTime=0i,token="A3" 1453801854151187370
QueueEvent,operation=2,queue=2,realm=1,user=5 queueLength=0i,waitTime=0i,token="B11" 1453801854151385268
QueueEvent,operation=1,queue=1,realm=1,user=3 queueLength=1i,token="A1" 1453801854167831645
QueueEvent,operation=1,queue=2,realm=1,user=3 queueLength=1i,token="B12" 1453801854183426408
QueueEvent,operation=1,queue=1,realm=2,user=2 queueLength=1i,token="C102" 1453801854292311968
QueueEvent,operation=1,queue=1,realm=2,user=2 queueLength=2i,token="C100" 1453801854307442083
QueueEvent,operation=1,queue=1,realm=1,user=3 queueLength=2i,token="A2" 1453801859317502431
QueueEvent,operation=1,queue=2,realm=1,user=3 queueLength=2i,token="B10" 1453801859442857888
QueueEvent,operation=4,queue=1,realm=1 queueLength=1i,token="A1" 1453801864906132158
QueueEvent,operation=4,queue=2,realm=1 queueLength=1i,token="B12" 1453801864906979599
QueueEvent,operation=2,queue=1,realm=1,user=5 queueLength=0i,waitTime=6i,token="A2" 1453801865452001148
QueueEvent,operation=2,queue=2,realm=1,user=5 queueLength=0i,waitTime=6i,token="B10" 1453801865467132101
QueueEvent,operation=2,queue=1,realm=2,user=2 queueLength=1i,waitTime=11i,token="C102" 1453801865483764954
QueueEvent,operation=4,queue=1,realm=2 queueLength=0i,token="C100" 1453801870912058022
```

**Comment by hoomanv** [comment]
I upgraded to 0.10.0-0.beta2 and then tried the query but received this error
```ERR: dial tcp: missing address```
I then did a clean install by deleting ```/var/lib/influxdb``` and reinserted my sample data and tried the query. This time it didn't fail with ```ERR: dial tcp``` however the result is still empty

**Comment by rossmcdonald** [comment]
@hoomanv Thank you for the extra information, and sample data. I've been able to reproduce the issue you're seeing.

@benbjohnson Do you know what the problem could be here?

**Comment by hoomanv** [comment]
Could it be related to having a few null ```queueLength``` fields?
It happens for ```waitTime``` field as well

**Comment by hoomanv** [comment]
@rossmcdonald @benbjohnson 
I figured that combination of ```min``` and ```count``` together will reproduce the problem.

**Comment by hoomanv** [comment]
@rossmcdonald
Currently the only workaround is to include ```fill(none)``` but since it is a continues query which imports the results into another measurement, it gets filled with unnecessary zeros.
Would it be possible to roll it in 0.11.0?

**Comment by hoomanv** [comment]
Appears to be fixed in 0.12.0
